src/wallet/spend.cpp

```diff
@@ -925,7 +958,7 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
     // behavior."
     const uint32_t nSequence{coin_control.m_signal_bip125_rbf.value_or(wallet.m_signal_rbf) ? MAX_BIP125_RBF_SEQUENCE : CTxIn::MAX_SEQUENCE_NONFINAL};
     for (const auto& coin : selected_coins) {
-        txNew.vin.push_back(CTxIn(coin.outpoint, CScript(), nSequence));
+        txNew.vin.push_back(CTxIn(coin->outpoint, CScript(), nSequence));
```

Comment:
could use emplace_back
```suggestion
        txNew.vin.emplace_back(coin->outpoint, CScript(), nSequence);
```

Reply:
agree but it's unrelated to the commit changes. Let's better leave it for another PR.